### PR TITLE
fix(protocol-designer): Fix deck view size issues in pd

### PR DESCRIPTION
**Pull request by koji**
# Overview
Fix deck view size issues in pd and fixing liquid name display.
this PR still keeps `outlines` because I'm waiting for Mel's final check. (OT-2 deck view) 
https://opentrons.slack.com/archives/C07E9T2NAQK/p1730217950806939?thread_ts=1730140254.488469&cid=C07E9T2NAQK
For Flex, Mel approved the chagnes.

Deck setup and Overview
Screenshot 2024-10-28 at 2 11 57 PM

with slot details
Screenshot 2024-10-28 at 3 19 12 PM

liquids overflow menu
Screenshot 2024-10-28 at 3 16 55 PM

Screenshot 2024-10-29 at 12 37 20 PM

updated the slot detail displaying position

https://github.com/user-attachments/assets/80e25e62-e44d-4f25-915c-e24cf304fe1b

https://github.com/user-attachments/assets/efb74e41-8100-4b26-a245-88013c2977d8

close RQA-3333 RQA-3435 AUTH-973 RQA-3379 RQA-3334

## Risk assessment
low

### CODE REVIEW

**Comment by koji** [comment]
- [x] remove outline for debugging
- [x] fix a long liquid name

File: components/src/hardware-sim/RobotCoordinateSpace/RobotCoordinateSpaceWithRef.tsx [hunk 1]
```diff
@@ -31,16 +33,36 @@ export function RobotCoordinateSpaceWithRef(
...
   return (
-    <Svg
-      viewBox={viewBox || wholeDeckViewBox}
-      ref={wrapperRef}
-      transform="scale(1, -1)"
-      {...restProps}
+    <Flex
+      width="100%"
+      height="100%"
+      alignItems="center"
```
Comment (jerader) [inline components/src/hardware-sim/RobotCoordinateSpace/RobotCoordinateSpaceWithRef.tsx]:
```suggestion
      alignItems={ALIGN_CENTER}
```

File: components/src/hardware-sim/RobotCoordinateSpace/RobotCoordinateSpaceWithRef.tsx [hunk 2]
```diff
@@ -31,16 +33,36 @@ export function RobotCoordinateSpaceWithRef(
...
+      justifyContent="center"
```
Comment (jerader) [inline components/src/hardware-sim/RobotCoordinateSpace/RobotCoordinateSpaceWithRef.tsx]:
```suggestion
      justifyContent={JUSTIFY_CONTENT_CENTER}
```

File: components/src/hardware-sim/RobotCoordinateSpace/RobotCoordinateSpaceWithRef.tsx [hunk 3]
```diff
@@ -31,16 +33,36 @@ export function RobotCoordinateSpaceWithRef(
...
+      css={css`
+        outline: purple solid 1px;
```
Comment (jerader) [inline components/src/hardware-sim/RobotCoordinateSpace/RobotCoordinateSpaceWithRef.tsx]: i think these outlines were to show design team easier?

**Review by jerader — COMMENTED** [review]
the deckmap looks really good! but the slot details when hovering over a slot needs to be realigned

Screenshot 2024-10-29 at 10 24 00

File: components/src/hardware-sim/RobotCoordinateSpace/RobotCoordinateSpaceWithRef.tsx (hunk 3 above)
Comment (koji) [reply components/src/hardware-sim/RobotCoordinateSpace/RobotCoordinateSpaceWithRef.tsx]: yup, I will remove outlines when I get approval from Mel.

**Comment by jerader** [comment]
looks good except for when there is a staging area

Screenshot 2024-10-30 at 15 21 05

oh, thanks I will modify the util function

**Review by jerader — APPROVED** [review]
lgtm

**Pull request merged**